spec/requests/notifications_spec.rb

```diff
@@ -344,12 +344,11 @@ def mock_heart_reaction_notifications(amount, categories, reactable = article1)
           body_markdown: "@#{user.username}",
         )
       end
+      let(:mention) { create(:mention, mentionable: comment, user: user) }
 
       before do
-        comment
-        perform_enqueued_jobs do
-          Mention.create_all(comment)
-          Notification.send_mention_notification(Mention.first)
+        Sidekiq::Testing.inline! do
```

Comment:
Ideally, I want to make a nice little testing helper for this but thank you for doing it here by hand!!!